operations/helm/charts/mimir-distributed/templates/query-frontend/query-frontend-so.yaml

```diff
@@ -24,14 +24,17 @@ spec:
     kind: Deployment
   triggers:
   - metadata:
-      metricName: query_frontend_cpu_hpa_default
+      query: max_over_time(sum(sum by (pod) (rate(container_cpu_usage_seconds_total{container="query-frontend",namespace="{{ .Release.Namespace }}"}[5m])) and max by (pod) (up{container="query-frontend",namespace="{{ .Release.Namespace }}"}) > 0)[15m:]) * 1000
       query: max_over_time(sum(rate(container_cpu_usage_seconds_total{container="query-frontend",namespace="{{ .Release.Namespace }}"}[5m]))[15m:]) * 1000
```

Comment:
the old query is still here, can you remove it? Same goes for the memory query below

Reply:
Copy and paste error, sorry for that